app/shared/utils/EmptinessChecker.scala

```diff
@@ -95,63 +95,67 @@ object EmptinessChecker {
     case object Null      extends Structure
   }
 
-  def apply[A](implicit aInstance: EmptinessChecker[A]): EmptinessChecker[A] = aInstance
+  def apply[A](using aInstance: EmptinessChecker[A]): EmptinessChecker[A] = aInstance
 
   def instance[A](func: A => Structure): EmptinessChecker[A] = (value: A) => func(value)
 
-  def instanceObj[A](func: A => Structure.Obj): ObjEmptinessChecker[A] = (value: A) => func(value)
-
-  def use[A, B: EmptinessChecker](func: A => B): EmptinessChecker[A] = EmptinessChecker.instance { a =>
-    val b = func(a)
-    EmptinessChecker[B].structureOf(b)
+  def use[A](func: A => List[(String, Structure)]): EmptinessChecker[A] = EmptinessChecker.instance { a =>
+    Structure.Obj(func(a))
   }
 
+  def field[A](name: String, value: A)(using checker: EmptinessChecker[A]): (String, Structure) = name -> checker.structureOf(value)
+
```

Comment:
`use` method updated and `field` method added for robust use of `EmptinessChecker`. This has been added to the reliefs POC.